src/plugins/intel_cpu/tests/functional/single_layer_tests/grid_sample.cpp

```diff
@@ -0,0 +1,343 @@
+            { { {}, { {1, 7, 5, 3} } },   // Static shapes
+              { {}, { {1, 1, 11, 2} } }
+            },
+            { { {}, { {2, 6, 7, 2} } },   // Static shapes
+              { {}, { {2, 6, 2, 2} } }
+            },
+            { { {}, { {3, 5, 6, 3} } },   // Static shapes
+              { {}, { {3, 1, 13, 2} } }
+            },
+            { { {}, { {4, 4, 5, 6} } },   // Static shapes
+              { {}, { {4, 2, 7, 2} } }
+            },
+            { { {}, { {5, 3, 4, 5} } },   // Static shapes
+              { {}, { {5, 3, 5, 2} } }
+            },
+            { { {}, { {4, 2, 7, 6} } },   // Static shapes
+              { {}, { {4, 4, 4, 2} } }
+            },
+            { { {}, { {3, 3, 9, 7} } },   // Static shapes
+              { {}, { {3, 1, 17, 2} } }
+            },
+            { { {}, { {2, 4, 9, 8} } },   // Static shapes
+              { {}, { {2, 19, 1, 2} } }
+            }
+        };
+        result.insert(result.end(), tmp.begin(), tmp.end());
+    } // AVX2
+
+    if (InferenceEngine::with_cpu_x86_avx512f()) {
+        std::vector<std::vector<InputShape>> tmp = {
+            { { {}, { {1, 7, 2, 9} } },    // Static shapes
+              { {}, { {1, 4, 5, 2} } }
+            },
+            { { {}, { {2, 6, 3, 10} } },   // Static shapes
+              { {}, { {2, 3, 7, 2} } },
+            },
+            { { {}, { {3, 5, 2, 11} } },   // Static shapes
+              { {}, { {3, 4, 6, 2} } }
+            },
+            { { {}, { {4, 4, 4, 12} } },   // Static shapes
+              { {}, { {4, 5, 5, 2} } },
+            },
+            { { {}, { {5, 3, 2, 13} } },   // Static shapes
+              { {}, { {5, 1, 31, 2} } },
+            },
+            { { {}, { {4, 3, 5, 14} } },   // Static shapes
+              { {}, { {4, 4, 8, 2} } },
+            },
+            { { {}, { {3, 2, 2, 15} } },   // Static shapes
+              { {}, { {3, 33, 1, 2} } },
+            },
+            { { {}, { {2, 1, 6, 16} } },   // Static shapes
+              { {}, { {2, 8, 8, 2} } },
+            },
+            { { {}, { {2, 3, 7, 17} } },   // Static shapes
+              { {}, { {2, 9, 9, 2} } },
+            }
+        };
+        result.insert(result.end(), tmp.begin(), tmp.end());
+    } // AVX512
+
+    return result;
+}
+
+INSTANTIATE_TEST_SUITE_P(smoke_static, GridSampleLayerTestCPU,
+                ::testing::Combine(
+                    ::testing::ValuesIn(getStaticShapes()),
+                    ::testing::ValuesIn(interpolateMode),
+                    ::testing::ValuesIn(paddingMode),
+                    ::testing::ValuesIn(alignCorners),
+                    ::testing::ValuesIn(dataPrecision),
+                    ::testing::ValuesIn(gridPrecision),
+                    ::testing::ValuesIn(getCPUInfo()),
+                    ::testing::ValuesIn(additionalConfig)),
+                GridSampleLayerTestCPU::getTestCaseName);
+
+
+const std::vector<std::vector<InputShape>> dynamicInSapes = {
```

Comment:
Could you please add a couple of tests with bounds for dynamic shapes?

Reply:
Which bounds do you mean?

Reply:
I mean that we can use undefined dimensions like `-1` and also we can use dimension bounds in dynamic cases like `[1..10]`. [Example ](https://github.com/openvinotoolkit/openvino/blob/349f3e02e020caf2e49aafa7b5ff8bffe1e31058/src/plugins/intel_cpu/tests/functional/single_layer_tests/pad.cpp#L532)
So I'd prefer to suggest both dynamic cases in tests

Reply:
There are already such cases on lines 297, 299, 305